developers/weaviate/client-libraries/typescript/v2_v3_migration.md

```diff
@@ -400,7 +400,158 @@ response.data?.Get?.Article?.[0]['_additional']?.creationTimeUnix // Get the tim
 
 ## How to migrate your code
 
-For code examples, see the pages here:
+
+This workflow describes the process one would follow to migrate their codebase to the v3 client. 
+
+1. Install the v3 client package. Update the client package from the v2 to v3 client. 
```

Comment:
Consider:

```
1. Install the v3 client package.
1. [Update the client package](link to page) from the v2 to v3 client.
```

Should this be something like, "Edit your code to import the v3 client package."